src/taco-utils/cordovaHelper.ts

```diff
@@ -341,6 +341,14 @@ module TacoUtility {
                 });
         }
 
+        public static getSavedPlugins(projectInfo: IProjectInfo): Q.Promise<Cordova.ICordovaPluginInfo[]> {
+            return TacoPackageLoader.lazyCordovaRequire(projectInfo.cordovaCliVersion)
```

Comment:
It seems to end up using cordova@ similar to doing:
npm install cordova@
And it seems to work.